script/server/index.ts

```diff
@@ -18,7 +18,7 @@ export default (options: Options): Promise<{server: http.Server, port: number}>
   const baseDir = path.join(__dirname, '../../build');
   const {port, onlyProxy} = {...defaultOptions, ...options};
 
-  if (!port) {
+  if (!port || !(port > 0)) {
```

Comment:
`port < 1` ?

Reply:
It was easier than writing `!port || !Number.isNAN(port) || port <= 0` . I ran into this area silently failing because `const port = parseInt(process.env.PORT || '', 10);` was set to `NAN`.

Reply:
the `not greater than 0` confused me. if your goal is to actually check for NaN i think we should do that explicitly